tests/storages_tests/rdb_tests/test_storage.py

```diff
@@ -183,7 +192,7 @@ def test_upgrade_identity() -> None:
     assert old_version == new_version
 
 
-@pytest.mark.parametrize("optuna_version", ["0.9.0", "1.2.0", "1.3.0", "2.4.0", "2.6.0"])
+@pytest.mark.parametrize("optuna_version", ["0.9.0", "1.2.0", "1.3.0", "2.4.0"])
```

Comment:
@toshihikoyanase 

I'm not confident but I think it would not change anything in the case where `optuna_version` is `2.6.0`.
(Before this PR, `storage.get_current_version()` and `storage.get_head_version()` are both `2.6.0`.

So I updated `2.6.0.db` to use in `test_upgrade_distributions` and `test_upgrade_{single,multi}_objective_optimization`.
dcf497deb

If I misunderstand something, please tell me.

Reply:
Certainly the current master does nothing with this test for `optuna_version="2.6.0"`. This test confirms that running the upgrade on 2.6.0 does not cause any errors at all. On the other hand, adding "2.6.0" here and creating a `test_upgrade_assets/2.6.0.db` with `optuna==2.6.0` will give us the following advantages when we need the storage migration in some Optuna version (e.g. 3.0.0).
- You can find out the version of Optuna that required migration in the past by looking at the code, without having to go through the PR on GitHub.
- You don't need to create a db file for the past optuna version, you can use it immediately from the code base.

Therefore, we should add "2.6.0" here in the past, and create db file with `optuna==2.6.0`.

Reply:
Sorry for not responding you, your comment totally make sense. I revised the PR to leave 2.6.0.db (albeit adding a new study) and add 3.0.0.db.